wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -301,12 +312,14 @@ def visit_any_assign(self, node: AnyAssign) -> None:
 
         Raises:
             ReassigningVariableToItselfViolation
+            ApproximateConstantViolation
 
         """
         names = list(name_nodes.flat_variable_names([node]))
 
         self._check_reassignment(node, names)
         self._check_unique_assignment(node, names)
+        self._check_approximate_constants(node, names)
```

Comment:
That's an incorrect visitor for this check. Use `builtins.py`. And as I said - we don't need names to be checked.